src/getting-started/tooling/probe-rs.md

```diff
@@ -1 +1,132 @@
-# probe-rs
+# `probe-rs`
+
+The [`probe-rs`][probe-rs] project is a set of tools to interact with embedded MCU's using various debug probes. It is similar to [OpenOCD][openocd], [pyOCD][pyocd], [Segger tools][segger-tools], etc. There is support for `Xtensa` & `RISC-V` architectures along with a collection of tools, including but not limited to:
+
+- Flashing algorithms
+- Debugger
+  - GDB support.
+  - CLI for interactive debugging.
+  - VS Code extension.
+- [Real Time Transfer (RTT)][rtt]
+  - Similar to [`app_trace` component of IDF][app-trace-idf].
+
+Espressif products containing the `USB-JTAG-SERIAL` peripheral can use `probe-rs` without any external hardware.
```

Comment:
Maybe mention that chips which _do not_ have this peripheral can still be used with an external programmer, e.g. an ESP-Prog?